Framework/DataHandling/src/LoadILLSANS.cpp

```diff
@@ -147,6 +147,33 @@ void LoadILLSANS::exec() {
     const double angle = firstEntry.getFloat(instrumentPath + "/Gamma/value");
     placeD16(-angle, distance, "detector");
 
+  } else if (m_instrumentName == "D11B") {
+    initWorkSpaceD11B(firstEntry, instrumentPath);
+    progress.report("Loading the instrument " + m_instrumentName);
+    runLoadInstrument();
+    double distance =
+        firstEntry.getFloat(instrumentPath + "/detector/det_calc") / 1000.;
+    moveDetectorDistance(distance, "detector_center");
+
+    distance =
```

Comment:
Unlike the D33, for those 2 instruments (to be confirmed by the scientists) the banks are on the same frame so they cannot move independently. It would be better if we fix the longitudinal offset in IDF, but in the loader just read the main detector distance and move the whole thing.

Reply:
I'm not sure we can just move the whole thing, but we could get the distance between the panel, move the main detector then move the panels to the main detector's distance + the offset. But I'm not sure if it would be a better way to do

Reply:
For D11B, the banks are put in a single component detector, so we can move just that.
I thought I made D22B similar, but apparently there is no parent component.

Reply:
Yes, and the parent component itself can only be moved using the relative position flag, so it is still a bit convoluted